doc/level0.rst

```diff
@@ -244,6 +256,60 @@ While the message structure is complex, it's actually fairly easy to
 pack and unpack with common OpenPGP tools.  It was selected to ease
 implementation and deployment, not for cleanliness or purity :)
 
+Example:
+
+::
+
+	To: me@mydomain.com
+	From: me@mydomain.com
+	Autocrypt-Setup-Message: v0
+	Content-type: multipart/mixed; boundary="==break0=="
+
+	--==break0==
+	Content-Type: text/plain
+
+	This is the Autocrypt setup message.
+
+	--==break0==
+	Content-Type: application/autocrypt-key-backup
+    Content-Disposition: attachment; filename="autocrypt-key-backup.html"
+
+	<html>
+	<body>
+	<p>
+	    This is the Autocrypt setup file used to transfer keys between clients.
+	</p>
+    <pre>
+    -----BEGIN PGP MESSAGE-----
+    Version: BCPG v1.53
+
+    hQIMAxC7JraDy7DVAQ//SK1NltM+r6uRf2BJEg+rnpmiwfAEIiopU0LeOQ6ysmZ0
+    CLlfUKAcryaxndj4sBsxLllXWzlNiFDHWw4OOUEZAZd8YRbOPfVq2I8+W4jO3Moe
+    -----END PGP MESSAGE-----
+    </pre>
+	</body>
+	</html>
+	--==break0==--
+
+The encrypted message part contains:
+
+::
+
+	Content-type: multipart/mixed; boundary="==break2=="
+	Autocrypt-Prefer-Encrypt: mutual
+
+	--==break2==
+	Content-type: application/autocrypt-key-backup
+
+	-----BEGIN PGP PRIVATE KEY BLOCK-----
+	Version: GnuPG v1.2.3 (GNU/Linux)
+
+	xcLYBFke7/8BCAD0TTmX9WJm9elc7/xrT4/lyzUDMLbuAuUqRINtCoUQPT2P3Snfx/jou1YcmjDgwT
+	Ny9ddjyLcdSKL/aR6qQ1UBvlC5xtriU/7hZV6OZEmW2ckF7UgGd6ajE+UEjUwJg2+eKxGWFGuZ1P7a
+	4Av1NXLayZDsYa91RC5hCsj+umLN2s+68ps5pzLP3NoK2zIFGoCRncgGI/pTAVmYDirhVoKh14hCh5
+	.....
+	-----END PGP PRIVATE KEY BLOCK-----
+	--==break2==--
```

Comment:
This example is great!  It currently duplicates the mesage in doc/transfer-format.rst, though.  we should have a single example -- one place or the other, and that example should actually be usable (as a test case).